examples/wasm-example/index.js

```diff
@@ -0,0 +1,92 @@
+/// The `@polkadot/extension-dapp` package can be dynamically imported.
+/// Usually it is wise to use a package manager like npm or yarn to install it as a dependency.
+/// The `getPolkadotJsExtensionMod`
```

Comment:
I'm not super familiar with js docs, but is this line complete here? Or should we remove it?